drivers/hmis/app/graphql/mutations/update_enrollment.rb

```diff
@@ -14,6 +14,8 @@ def resolve(id:, entry_date: nil, relationship_to_ho_h: nil)
       if enrollment
         enrollment.entry_date = entry_date if entry_date.present?
         enrollment.relationship_to_ho_h = relationship_to_ho_h if relationship_to_ho_h.present?
+        enrollment.date_updated = DateTime.current
```

Comment:
I'm all for doing this intentionally, but doesn't it happen automatically? Huh, I thought we'd specified which columns were the timestamp columns in the HUD tables, but maybe not.

Reply:
It didn't seem like it happened automatically to me. If you take this out I think the HMIS update tests will fail because the timestamps don't change. Maybe this was done on the warehouse models but not the HMIS ones?

Reply:
Yeah, I can't find anything that would cause it to happen automatically.  Thanks for adding.  Maybe we can re-factor those in the future.  This looks somewhat promising, but seems to be missing from rails 7 maybe?
https://apidock.com/rails/v6.1.3.1/ActiveRecord/Timestamp/ClassMethods/timestamp_attributes_for_create